### ISSUE TITLE
Use of distinct names for different artifacts

### ISSUE THREAD

**Issue by amullarney** [johnrwolfe/CarPark#70]
Use of identical names in different contexts can be confusing. It may also inhibit management of multiple versions of deployment configurations.  It would help to minimize unwanted returns from text string searches.

Suggestions:

Names of "shell" components used to provide non-xtUML representation of xtUML classes should use the class name with a suffix to be determined by convention. For example, from Carpark, EntryStandUI provides for GUI interaction with EntryStand instance(s).

Interface definitions might also be suffixed in this way - EntryStandIF.

Domain functions defined in "shell" components and invoked from port activities could use a DF suffix on the corresponding port signal name.

**Comment by SoftImp** [comment]
Agree!  Perhaps adding an underscore to indicate contextual relevance more clearly - e.g., EntryStand_UI and EntryStand_IF.
